administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -14,15 +14,23 @@
 JHtml::_('formbehavior.chosen', 'select');
 
 $this->app->getDocument()->addScriptDeclaration("
-	function iframeRef( frameRef ) {
-	return frameRef.contentWindow
-		? frameRef.contentWindow.document
-		: frameRef.contentDocument
-	}
+	function triggerSave(frame) {
+		if (frame == 'reference') {
+			window.frames[0].Joomla.submitbutton('" . $this->associatedView . ".apply');
+		}
+		if (frame == 'target') {
+			window.frames[1].Joomla.submitbutton('" . $this->associatedView . ".apply');
```

Comment:
no, why it would? i don't know. but it's possible to create an admin component and inject some kind of iframe in all admin pages,

i'm not saying that this a huge problem.
it's a good practice to not use array indexes in a flexible app like a cms is, when you can use direct id/name references.